lexos/static/js/scripts_dendrogram.js

```diff
@@ -68,12 +68,14 @@ function submitForm () {
  * @returns {string | null} - if it is null, it means no error, else then the string is the error message
  */
 function submissionError () {
-    const active_file_num_too_few_err = 'A dendrogram requires at least 2 active documents to be created.'
+    const active_file_num_too_few_err = 'You do not have enough active documents. Please activate at least two documents using the <a href="./manage">Manage</a> tool or <a href="./upload">upload</a> a new document.'
```

Comment:
Please use `url_for` here as well

Reply:
Using url_for provides links to 404 error pages. Not totally sure why.

Reply:
Fixed.